docs/dev_guide/pr_review_procedure.rst

```diff
@@ -35,17 +35,121 @@ It is important that approval for merging the PR is done on the comment thread,
 Continuous Integration
 ======================
 
-Currently we have a variety of services that respond or activate on an opened pull request:
+Currently we have a variety of services that respond or activate on an opened pull request.
 
-* `pep8speaks <https://github.com/OrkoHunter/pep8speaks>`_: Performs a PEP8 check on any submitted code.
+Comments from bots:
 
-* `CircleCi <https://circleci.com/gh/sunpy/sunpy/>`_: Tests to see if sunpy installs, builds the documentation and runs the figure tests.
+* `pep8speaks <https://github.com/OrkoHunter/pep8speaks>`_: Performs a PEP8 check on any submitted code. This is updated as the code changes.
 
-* Giles: Returns a link if the documentation builds successfully.
+Checks that  appear at the bottom of a pull request:
 
-* `Azure Pipelines <https://dev.azure.com/sunpy/sunpy/_build>`_: Runs our test suite on all three operating systems.
+.. image:: images/checks_pr.png
+   :width: 600
+   :alt: PR checks
 
-* `CodeCov <https://codecov.io/gh/sunpy/sunpy/>`_: Checks how many lines of the code lack test coverage.
+or at the top under the "Checks" tab:
+
+.. image:: images/checks.png
+   :width: 600
+   :alt: PR checks tab
+
+* `figure-tests (CircleCi) <https://circleci.com/gh/sunpy/sunpy/>`_: Runs two figure tests environments ("py38-figure", "py37-figure-devdeps").
+
+* figure_report/figure_tests (Giles): Show the final results of the figure tests.
+
+* figure_report_devdeps (Giles): Show the final results of the figure tests using development packages.
+
+* changelog: absent | found (Giles): If a changelog is needed, this will check and will pass if a changelog with the correct number is found.
+
+* milestone: absent | present (Giles): Will check that the pull request has a milestone assigned.
+
+* `docs/readthedocs.org:sunpy (Read the Docs) <https://readthedocs.org/projects/sunpy/>`_: This builds our documentation.
+
+* `sunpy.sunpy (Azure Pipelines) <https://dev.azure.com/sunpy/sunpy/_build>`_: Runs our test suite on all three operating systems.
+  There are 10 separate checks for this.
+
+* `codecov/patch (CodeCov) <https://codecov.io/gh/sunpy/sunpy/>`_: Checks how many lines of the code lack test coverage for the submitted code.
+
+* `codecov/project (CodeCov) <https://codecov.io/gh/sunpy/sunpy/>`_: Checks how many lines of the code lack test coverage in sunpy overall.
+
+It is common to see some of these checks fail.
+This can be happen to due a change that has broken a test or a remote server that we use has failed.
+Therefore it is important to check why a task failed and if has a pre-existing issue, it can be safe to ignore on that pull request.
```

Comment:
```suggestion
Therefore it is important to check why a task failed and if has a pre-existing issue, it can be safe to ignore a failing check on that pull request.
```